python/README.md

```diff
@@ -4,11 +4,40 @@ This library provides a Pythonic API wrapper for the reference Arrow C++
 implementation, along with tools for interoperability with pandas, NumPy, and
 other traditional Python scientific computing packages.
 
-#### Development details
+### Development details
 
 This project is layered in two pieces:
 
 * pyarrow, a C++ library for easier interoperability between Arrow C++, NumPy,
   and pandas
 * Cython extensions and pure Python code under arrow/ which expose Arrow C++
-  and pyarrow to pure Python users
\ No newline at end of file
+  and pyarrow to pure Python users
+
+#### PyArrow Dependencies:
+These are the various projects that PyArrow depends on.
+
+1. **g++ and gcc Version >= 4.9**
```

Comment:
Minimal GCC is 4.8, working locally and in the `manylinux1` docker image. GCC 4.7 is definitely not working as we're using `using` features implemented first in 4.8.